README.md

```diff
@@ -1,6 +1,9 @@
 # RHOAS CLI
+<p align="center">
```

Comment:
That works but was replaced by css that looks like this.
No strong opinion to change it but it simply looks better :D
```suggestion
<p style="width:400px; text-align:center;">
```

Reply:
Actually entire container should be centered and p tag can be omitted

Reply:
What would be the functional benefit of changing this?

Reply:
controling with of the content depending on the size of the screen - so image is not stretched on 16:9 screens etc.

Reply:
Igore me. This is readme. Doesn't matter much